sentry-android-replay/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:tools="http://schemas.android.com/tools">
+
+  <!-- Compose requires min SDK 21, but our min SDK is set to 19 -->
+  <uses-sdk
```

Comment:
just a thought - shall we already target the v8 branch? 🤔 unless we want to rush this PR, I guess it will be easier actually?

Reply:
Yes, we could - let me try!